docsource/modules160-170.rst

```diff
@@ -916,7 +916,7 @@ Module coverage 16.0 -> 17.0
 +---------------------------------------------------+----------------------+-------------------------------------------------+
 | sale_purchase_stock                               | Nothing to do        |                                                 |
 +---------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| sale_quotation_builder                      |                      |Merged into sale_pdf_quote_builder.              |
+| |del| sale_quotation_builder                      | Done                 |Forward-ported into OCA/sale-reporting           |
```

Comment:
We should leave blank the past of "Done", do the module is not installed in the DB for the CI.